datakyt/db/insert_data_in_project_table.py

```diff
@@ -0,0 +1,27 @@
+import psycopg2
+import csv
+import os
+
+
+def insert_csv_file_to_database(path, conn):
+    cur = conn.cursor()
+    with open(os.path.join(path), 'r') as f:
+        reader = csv.reader(f)
+        next(reader)
+        cont_of_csv = []
+        for row in reader:
+            cur.execute('INSERT INTO PROJECT VALUES(%s,%s)',
+                        row)
+            cont_of_csv.append([int(row[0]), row[1]])
+            conn.commit()
+        cur.close()
+        conn.close()
+
+    return cont_of_csv
```

Comment:
why do we return csv?

Reply:
it is an array where I collected all rows, so it is not csv file

Reply:
it is not clear why we return something at all?